package.json

```diff
@@ -43,17 +44,22 @@
     "truffle-hdwallet-provider": "~1.0.17"
   },
   "devDependencies": {
-    "@truffle/contract": "^4.2.23",
+    "@openeth/truffle-typings": "0.0.6",
```

Comment:
Are we sure we could move this dependency into `devDependencies`? What about the libraries that depend on the `rif-relay-contracts`? I'm afraid that they won't install it

Reply:
the library is installed twice, it its in the dependencies too, therefore I removed from the dev.